src/test/java/com/ericsson/ei/subscriptionhandler/test/SubscriptionValidatorTest.java

```diff
@@ -16,20 +16,20 @@
 */
 package com.ericsson.ei.subscriptionhandler.test;
```

Comment:
Not sure but it seems this file has not been changed much, other than perhaps indentation? Double check that it only spaces and not tabs :)

Reply:
A new test is added in this file, validateSubscriptionWithSchemaTest()

indentation is spaces